website/cue/reference/components/sources/base/http_client.cue

```diff
@@ -515,7 +515,7 @@ base: components: sources: http_client: configuration: {
 			options: "*": {
 				description: "A query string parameter and it's value(s)."
 				required:    true
-				type: array: items: type: string: {}
```

Comment:
Technically this should indicate that either a `string` or `[string]` can be used, but I'm not sure the current cue schema supports this (or if the code that generates the cue from the struct definitions does) 🤔